epistles-v1/original/04-epistles.adoc

```diff
@@ -548,8 +552,8 @@ G+++.+++ F.
 [.blurb]
 ==== To Friends in the North.
 
-All Friends in Westmoreland, Cumberland, Bishopric, New Castle-side, Northumberland,
-and Lancashire,
+[.blurb]
+=== All Friends in Westmoreland, Cumberland, Bishopric, New Castle-side, Northumberland, and Lancashire,
```

Comment:
Should this be a blurb? I didn't know what to do with it. It didn't look right as a salutation.

Reply:
yeah, i tried it a couple ways too... i didn't really love any of them, but I think i lean towards making this a _salutation_ instead.